packages/odyssey/src/scss/components/_table.scss

```diff
@@ -26,19 +26,15 @@
   }
 
   caption {
-    margin-bottom: $small-spacing;
-    color: $text-heading;
-    font-size: ms(1);
-    font-weight: 600;
-    text-align: left;
+    @include is-visually-hidden;
```

Comment:
for my education: why is this different than the class name `.m-visually-hidden` shouldn't the class name and the mixin name be consistent?

Reply:
^ unintentional. it's a holdover from our original codebase - we should definitely set some better standards around naming.